pkg/orchestration/wiring/testdata/ec2compute.customrdsreplica.bundle.yaml

```diff
@@ -87,7 +84,7 @@ spec:
               env: microstestenv
             resource_name: db
             shareddb:
-              resource: '!{rds--instance-metadata-name}'
```

Comment:
It used to pull in the name of the `ServiceInstance` but now uses the name of the Resource in the State. Both are `rds` strings, but semantically not the same thing. This is to match what RDS autowiring function does.